src/Datadog.Trace/Tracer.cs

```diff
@@ -682,7 +714,17 @@ private void CurrentDomain_UnhandledException(object sender, UnhandledExceptionE
 
         private void Console_CancelKeyPress(object sender, ConsoleCancelEventArgs e)
         {
-            RunShutdownTasks();
+            // Console Cancel KeyPress is raised on SIGINT and SIGQUIT signal,
```

Comment:
This is also raised when user presses CTRL+C in a Windows console app :) 
Did we test that this and `ProcessExit` still interact correctly?

Reply:
I’ll add the comment about CTRL+C. About the test, see my comment below.

Reply:
I've improved the comment in d39e660